### PR TITLE
Endpoints for SystemEarse, LC Wipe, Backup Image and restore Image.

### PR DESCRIPTION
**Pull request by prashanthlgowda**
Endpoints for SystemEarse, LC Wipe, Backup Image and restore Image.

### CODE REVIEW

File: src/main/java/com/dell/isg/smi/service/server/configuration/manager/ConfigurationManagerImpl.java [hunk 1]
```diff
@@ -154,8 +157,79 @@ public XmlConfig exportInventory(ServerAndNetworkShareRequest request) throws Ex
...
+	@Override
+	public XmlConfig wipeLifeController(Credential request) throws Exception {
+		WsmanCredentials wsmanCredentials = new WsmanCredentials(request.getAddress(), request.getUserName(),
+				request.getPassword());
+		XmlConfig config = configAdapter.wipeLifeController(wsmanCredentials);
+		return config;
+	}
+
+	@Override
+	public XmlConfig systemEarseServer(SystemEarseRequest request) throws Exception {
```
Comment (muqeethkowkab) [inline src/main/java/com/dell/isg/smi/service/server/configuration/manager/ConfigurationManagerImpl.java]: Can you fix the typo.  SystemEraseRequest

File: src/main/java/com/dell/isg/smi/service/server/configuration/controller/ConfigurationController.java [hunk 2]
```diff
@@ -285,6 +291,162 @@
...
+    /**
+     * Endpoint to create granular, user selectable, categories to increase flexibility and improve the repurposing aspect of the existing System Wipe feature.
+     * @param request - Export configuration request.
+     * @param bindingResult - Export configuration response binder.
+     * @return
+     * @throws Exception - On failure 
+     */
+    @RequestMapping(value = "/systemEarse", method = RequestMethod.POST, headers = "Accept=application/json", consumes = "application/json", produces = "application/json")
+    @ApiOperation(value = "System Earse", nickname = "import", notes = "This operation allow user to create granular, user selectable, categories to increase flexibility and improve the repurposing aspect of the existing System Wipe feature. List of components can be BIOS_RESET_DEFULT, EMBEDDED_DIAGNOSTICS_ERASE, OS_DRIVERPACK_ERASE, IDRAC_DEFAULT and LC_DATA_ERASE", response = ServiceResponse.class)
+    public ResponseEntity<ServiceResponse> systemEarse(@RequestBody @Valid SystemEarseRequest request, BindingResult bindingResult) throws Exception {
```
Comment (muqeethkowkab) [inline src/main/java/com/dell/isg/smi/service/server/configuration/controller/ConfigurationController.java]: Can you fix the typo. SystemEraseRequest

File: src/main/java/com/dell/isg/smi/service/server/configuration/validators/SystemEarseValidator.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+import com.dell.isg.smi.commons.model.common.Credential;
+import com.dell.isg.smi.service.server.configuration.model.SystemEarseRequest;
+
+/**
+ * @author Muqeeth_Kowkab
+ *
+ */
+@Component
+public class SystemEarseValidator implements Validator {
```
Comment (muqeethkowkab) [inline src/main/java/com/dell/isg/smi/service/server/configuration/validators/SystemEarseValidator.java]: SystemEraseValidator typo change

**Review by muqeethkowkab — APPROVED** [review]
Looks Good. Can you fix the typo before pushing..

**Comment by prashanthlgowda** [comment]
Done the changes,

**Review by rahman-muhammad — APPROVED** [review]

**Pull request merged**